lib/categories.nix

```diff
@@ -0,0 +1,949 @@
+/* List of categories
+    ```nix
+    handle = {
+      # Required
+      name = "Category Name";
+      description = ''
+        Description of category
+      '';
+      relatedCategories = [ related01 related02 . . . ];
```

Comment:
I think these will be parent categories, in the end; it might end up being a DAG not a forest.

I agree, though, that cutting the not-strictly-necessary stuff at first is often good, and adding the relation records will not become harder with time.